Dockerfile

```diff
@@ -90,6 +90,33 @@ RUN apt-get purge -y --auto-remove -o APT::AutoRemove::RecommendsImportant=false
 
 CMD ["/bin/bash"]
 
+# `docs_italia_it_celery_web`: Build image for celery-web
+# We need additional packages to run the document converter
+
+FROM docs_italia_it_web AS docs_italia_it_celery_web
+
+RUN apt-get update && apt-get install -y --no-install-recommends \
+        unzip \
```

Comment:
forse `unzip` possiamo includerlo nell'immagine principale? Il peso che aggiunge tenerselo secondo me è più che compensato dal processo di build più rapido

Reply:
Fixed in https://github.com/italia/docs.italia.it/pull/287/commits/9743d0e9e412c7035355858013e3ebdb44535b65